examples/function/flatmap_stream/example.py

```diff
@@ -0,0 +1,22 @@
+import aiorun
+from collections.abc import AsyncIterable
+
+from pynumaflow.function import Message, Datum, AsyncServer
+
+
+async def map_stream_handler(_: list[str], datum: Datum) -> AsyncIterable[Message]:
+    val = datum.value
```

Comment:
Let's add a docstring here maybe? Briefly explaining what this udf is trying to do with the streaming data